sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -231,6 +237,21 @@ CosmosClientBuilder setApiType(ApiType apiType){
         return this;
     }
 
+    /**
+     * Adds a policy for modifying request options dynamically.
+     *
```

Comment:
I'd call out that the last policy would win out on the override.

Reply:
added extra sentence calling this out.